### PR TITLE
Ddce 874

### PR DESCRIPTION
**Pull request by prameya-phatak**
# DDCE-(874)

##New feature

Not Eligible 3 Page

## PR Suggestions
- Have you looked at the JIRA story to make sure all Acceptance Criteria has been met?
- Have you done a visual check of the changes?
- Is there any still commented or unused code? Lingering printlns?
- Have things been implemented in a complicated way?
- Are the test still over the coverage threshold?
- Does the compiler throw a lot of warnings? 
- Have methods and tests been named clearly?
- Were there any changes in the config? Do changes need to be made in app-config-???

## Checklist PR Raiser
 - [x]  I've made every effort to commit high quality, clean code and I have executed relevant static analyses to be sure
 - [x]  I've included appropriate tests with any code I've added (Unit, Integration, Acceptance etc.)
 - [x]  I've executed the acceptance test pack locally to ensure there are no functional regressions (Performance tests where relevant)
 - [x]  I've squashed my commits - including the JIRA issue number in the commit message
 - [ ]  I've run a dependency check to ensure all dependencies are up to date

## Checklist PR Reviewer
 - [x]  I've made every effort to commit high quality, clean code and I have executed relevant static analyses to be sure
 - [ ]  I've included appropriate tests with any code I've added (Unit, Integration, Acceptance etc.)
 - [x]  I've executed the acceptance test pack locally to ensure there are no functional regressions (Performance tests where relevant)
 - [x]  I've squashed my commits - including the JIRA issue number in the commit message
 - [ ]  I've run a dependency check to ensure all dependencies are up to date

### CODE REVIEW

File: app/controllers/checkEligibility/InEligibleLocationOtherController.scala [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+import controllers.LocalBaseController
+import controllers.actions._
+import javax.inject.Inject
+import navigation.EligibilityNavigator
+import play.api.mvc._
+import repositories.SessionRepository
+import views.html.checkEligibility.InEligibleLocationOtherView
+
+class InEligibleLocationOtherController @Inject()(
+    val sessionRepository: SessionRepository,
```
Comment (abhijeetardale) [inline app/controllers/checkEligibility/InEligibleLocationOtherController.scala:29]: Unit tests are missing

File: app/views/checkEligibility/InEligibleLocationOtherView.scala.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+@govukLayout(pageTitle = Some(titleNoForm("notEligible.title", None)), signOut = false) {
+
+    @h1(messages("notEligible.heading"))
+
+    @p(Html(messages("notEligible.locationOther.p1")))
+    @p(Html(messages("notEligible.p2")))
+    @p{
+        @messages("notEligible.p3")
+        @link("https://www.gov.uk/charities-and-tax/get-recognition",messages("notEligible.p3.link"), isExternal = true)
```
Comment (abhijeetardale) [inline app/views/checkEligibility/InEligibleLocationOtherView.scala.html:38]: can we move this  link to application.conf and then read it using appConfig please?

File: app/views/checkEligibility/InEligibleCharitablePurposesView.scala.html [hunk 3]
```diff
@@ -27,14 +27,14 @@
...
 
-    @p(Html(messages("inEligibleCharitablePurposes.p1")))
-    @p(Html(messages("inEligibleCharitablePurposes.p2")))
+    @p(Html(messages("notEligible.charitablePurposes.p1")))
+    @p(Html(messages("notEligible.p2")))
     @p{
-        @messages("inEligibleCharitablePurposes.p3")
-        @link("https://www.gov.uk/charities-and-tax/get-recognition",messages("inEligibleCharitablePurposes.p3.link"), isExternal = true)
+        @messages("notEligible.p3")
+        @link("https://www.gov.uk/charities-and-tax/get-recognition",messages("notEligible.p3.link"), isExternal = true)
```
Comment (abhijeetardale) [inline app/views/checkEligibility/InEligibleCharitablePurposesView.scala.html:38]: can we move this link to application.conf and then read it using appConfig please? its applicable for other inEligible pages

File: app/controllers/checkEligibility/InEligibleLocationOtherController.scala (hunk 1 above)
Comment (prameya-phatak) [reply app/controllers/checkEligibility/InEligibleLocationOtherController.scala:29]: added test

**Pull request merged**